### PR TITLE
add new plugin 'kuota-calc'

### PR DESCRIPTION
**Pull request by ghouscht**
Hi there,
First of all thanks for maintaining krew - I really like it!

Wanted to add our `kuota-calc` plugin to the index. The plugin will soon receive some polishing and new resources/features will be added to support more resources.

Regards, Thomas

### CODE REVIEW

**Comment by ghouscht** [comment]
FYI: release automation is already prepared in branch [krew-release-update](https://github.com/postfinance/kuota-calc/tree/krew-release-update)

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/kuota-calc.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kuota-calc
+spec:
+  version: v0.1.0
+  homepage: https://github.com/postfinance/kuota-calc
+  shortDescription: Calculate the maximum needed resource quota for k8s objects.
```
Comment (ahmetb) [inline plugins/kuota-calc.yaml:8]: typically it's not great to include words like `k8s` here as it's obvious.
you could rewrite as:

> Calculate needed space to perform rolling updates

but you can fix it later.

**Comment by ahmetb** [comment]
Sorry for the late review. I think the shortdescription/description can be made better over time.
I'll merge this for now, please do improve.
/lgtm
/approve
🎄

File: plugins/kuota-calc.yaml (hunk 1 above)
Comment (ghouscht) [reply plugins/kuota-calc.yaml:8]: Will do, thank you!

**Comment by ghouscht** [comment]
> Sorry for the late review. I think the shortdescription/description can be made better over time. I'll merge this for now, please do improve. /lgtm /approve 🎄

No problem, thank you for merging, will adapt the description a bit. Wish you a merry Christmas. 🎄

**Pull request merged**